yt_dlp/extractor/globo.py

```diff
@@ -137,8 +137,8 @@ def _real_extract(self, url):
             formats, subtitles = self._extract_m3u8_formats_and_subtitles(
                 main_source['url'], video_id, 'mp4', m3u8_id='hls')
         self._merge_subtitles(traverse_obj(main_source, ('text', ..., {
```

Comment:
should we branch and try for both?
```suggestion
        self._merge_subtitles(traverse_obj(main_source, ('text', ..., ('caption', 'subtitle'), {
```

Reply:
ooo, into it, updated :+1: